src/export-plan/add-fonts.ts

```diff
@@ -0,0 +1,16 @@
Qcih0EHWYdjB6GHsQfYB/YH/ggIiAwIM4hGiFqIbYiHCI8InoixCLmIy4jUCPIJEolBiV4JbIl7iYsJqwnGieKJ94ocCi4KPwpRCm2Kd4qCCo2KoQq7CtuK84sMCyULTAtwC3iLmIuqC7wLzwvsC/qMDQw+jF6Mf4yhDM2M+A0jjVENcY2ODaqNx43tjfeOAg4NjiEOPw5hDniOj46njs0O8A7/jx0PMY9Gj10Pe4+Lj6IPvg/gEAsQIxAtEEIQQhBCEEIQQhBCEEIQQhBCEEIQQhBCEEWQSRBMkFKQWJBokHeQhpChkLoQ0pDdkPMQ8xD6kQORA5ElET4RRhFnkYERsRHZgAAAAEAAADmAFAABQAAAAAAAgABAAIAFgAAAQABLQAAAAAAAAAQAMYAAwABBAkAAABuAAAAAwABBAkAAQAeAG4AAwABBAkAAgAOAIwAAwABBAkAAwBQAJoAAwABBAkABAAuAOoAAwABBAkABQBOARgAAwABBAkABgAmAWYAAwABBAkABwBaAYwAAwABBAkACQAaAeYAAwABBAkACwA2AgAAAwABBAkADAA2AjYAAwABBAkAEAAYAmwAAwABBAkAEQAOAoQAAwABBAkAEgAoApIAAwABBAkAyAAWAroAAwABBAkAyQAwAtAAQwBvAHAAeQByAGkAZwBoAHQAIAAoAGMAKQAgAE0AYQByAGsAIABTAGkAbQBvAG4AcwBvAG4ALAAgADIAMAAwADUALgAgAEEAbABsACAAcgBpAGcAaAB0AHMAIAByAGUAcwBlAHIAdgBlAGQALgBQAHIAbwB4AGkAbQBhACAATgBvAHYAYQAgAFIAZwBSAGUAZwB1AGwAYQByAE0AYQByAGsAUwBpAG0AbwBuAHMAbwBuADoAIABQAHIAbwB4AGkAbQBhACAATgBvAHYAYQAgAFIAZQBnAHUAbABhAHIAOgAgADIAMAAwADUAUAByAG8AeABpAG0AYQAgAE4AbwB2AGEAIABSAGcAIABSAGUAZwB1AGwAYQByAFYAZQByAHMAaQBvAG4AIAAxAC4AMQAwADEAOwBQAFMAIAAwADAAMQAuADAAMAAxADsAaABvAHQAYwBvAG4AdgAgADEALgAwAC4AMwA4AFAAcgBvAHgAaQBtAGEATgBvAHYAYQAtAFIAZQBnAHUAbABhAHIAUAByAG8AeABpAG0AYQAgAE4AbwB2AGEAIABpAHMAIABhACAAdAByAGEAZABlAG0AYQByAGsAIABvAGYAIABNAGEAcgBrACAAUwBpAG0AbwBuAHMAbwBuAC4ATQBhAHIAawAgAFMAaQBtAG8AbgBzAG8AbgBoAHQAdABwADoALwAvAHcAdwB3AC4AbQBhAHIAawBzAGkAbQBvAG4AcwBvAG4ALgBjAG8AbQBoAHQAdABwADoALwAvAHcAdwB3AC4AbQBhAHIAawBzAGkAbQBvAG4AcwBvAG4ALgBjAG8AbQBQAHIAbwB4AGkAbQBhACAATgBvAHYAYQBSAGUAZwB1AGwAYQByAFAAcgBvAHgAaQBtAGEAIABOAG8AdgBhACAAUgBlAGcAdQBsAGEAcgBXAGUAYgBmAG8AbgB0ACAAMQAuADAAVABoAHUAIABNAGEAeQAgADMAMQAgADEAMAA6ADEAMAA6ADAAMAAgADIAMAAxADIAAAACAAAAAAAA/wUAKAAAAAAAAAAAAAAAAAAAAAAAAAAAAOYAAAECAQMAAwAEAAUABgAHAAgACQAKAAsADAANAA4ADwAQABEAEgATABQAFQAWABcAGAAZABoAGwAcAB0AHgAfACAAIQAiACMAJAAlACYAJwAoACkAKgArACwALQAuAC8AMAAxADIAMwA0ADUANgA3ADgAOQA6ADsAPAA9AD4APwBAAEEAQgBEAEUARgBHAEgASQBKAEsATABNAE4ATwBQAFEAUgBTAFQAVQBWAFcAWABZAFoAWwBcAF0AXgBfAGAAYQEEAKMAhACFAL0AlgDoAIYAjgCLAJ0AqQCkAQUAigCDAJMBBgEHAI0AiADeAQgAngCqAPUA9AD2AKIArQDJAMcArgBiAGMAkABkAMsAZQDIAMoAzwDMAM0AzgDpAGYA0wDQANEArwBnAPAAkQDWANQA1QBoAOsA7QCJAGoAaQBrAG0AbABuAKAAbwBxAHAAcgBzAHUAdAB2AHcA6gB4AHoAeQB7AH0AfAC4AKEAfwB+AIAAgQDsAO4AugCwALEAuwDYANkBCQEKAQsBDAENAQ4BDwEQAREBEgETARQBFQEWALIAswC2ALcAxAC0ALUAxQCHAKsBFwC+AL8BGAEZAIwBGgEbARwBHQEeBmdseXBoMQZnbHlwaDIHdW5pMDBBMAd1bmkwMEFEB3VuaTAwQjIHdW5pMDBCMwd1bmkwMEI5B3VuaTIwMDAHdW5pMjAwMQd1bmkyMDAyB3VuaTIwMDMHdW5pMjAwNAd1bmkyMDA1B3VuaTIwMDYHdW5pMjAwNwd1bmkyMDA4B3VuaTIwMDkHdW5pMjAwQQd1bmkyMDEwB3VuaTIwMTEKZmlndXJlZGFzaAd1bmkyMDJGB3VuaTIwNUYERXVybwd1bmlFMDAwB3VuaUZCMDEHdW5pRkIwMgd1bmlGQjAzB3VuaUZCMDS4Af+FsAGNAEuwCFBYsQEBjlmxRgYrWCGwEFlLsBRSWCGwgFkdsAYrXFgAsAMgRbADK0SwCCBFsgNKAiuwAytEsAcgRbIIPQIrsAMrRLAGIEWyBzACK7ADK0SwBSBFsgYiAiuwAytEsAQgRboABQEUAAIrsAMrRLAJIEWyAxQCK7ADK0QBsAogRbADK0SwCyBFsgpJAiuxA0Z2K0SwDCBFsgtqAiuxA0Z2K0SwDSBFsgxyAiuxA0Z2K0SwDiBFsg1SAiuxA0Z2K0SwDyBFsg47AiuxA0Z2K0SwECBFsg8hAiuxA0Z2K0SwESBFshAeAiuxA0Z2K0RZsBQrAAAAAU/He7gAAA==';
```

Comment:
What is this font? Can it be not hard-coded in this ts file?

Reply:
Yeah unfortunately this is how you add fonts in jspdf.... its the proxima nova one thats used elsewhere as well, but I need to add it as the base 64 version, and I need to use the ttf file version instead of the otf files that are in the assets folder already.

Reply:
Hmmm can it be read from a file or something?

Reply:
i guess this is the file for the font? I don't see any point in making it async.